app/views/layouts/application.html.erb

```diff
@@ -64,9 +64,9 @@
     <% end %>
 
     <% if alert.present? %>
-        <p class="notification">
-            <%= alert %>
-        </p>
+      <div class="error-summary" role="group" aria-labelledby="error-summary-heading" tabindex="-1">
```

Comment:
How does this relate to booking to nomis? Can you refactor this in a separate PR. Also I can see any unit test for this change.